cmd/stack/deploy.go

```diff
@@ -42,6 +52,12 @@ func Deploy(ctx context.Context) *cobra.Command {
 				}
 			}
 
+			composeEnv, present := os.LookupEnv(model.ComposeFile)
```

Comment:
@jLopezbarb what would be a good way to test this?

Reply:
We can encapsulate this code into a function that receives stackPath as parameter and check what happen if:
1. Stackpath is empty and compose file is set
2. Stackpath is empty and compose file is set with more than one path
3. Stackpath is set and compose file is set

Reply:
Doesn't this block of code doesn't cover the points 1 and 2 🤔 ?

```go
if len(options.StackPath) == 0 && present {
				options.StackPath = splitComposeFileEnv(composeEnv)
			}

Reply:
I'll rewrite as a function and update PR.